en/console-and-shells.rst

```diff
@@ -354,6 +354,19 @@ By default, a ``requested`` extra param is automatically added when a Shell
 is dispatched using ``dispatchShell()``. This ``requested`` parameter prevents
 the CakePHP console welcome message from being displayed on dispatched shells.
 
+Exit Status Codes
+-----------------
+It can sometimes be useful to return exit status codes. Simply use php's ``exit()`` statement.
```

Comment:
I don't think we should recommend putting `exit()` in console shells. Using `exit()` directly means it is impossible to test the containing method.  By default shell command methods that return `false` will emit a exit code of 1. If more exit codes are required, the `_stop()` method should be used. It wraps `exit()` and allows for mocking in test scenarios.